homeassistant/components/home_connect/strings.json

```diff
@@ -108,6 +108,10 @@
       "title": "Deprecated binary door sensor detected in some automations or scripts",
       "description": "The binary door sensor `{entity}`, which is deprecated, is used in the following automations or scripts:\n{items}\n\nA sensor entity with additional possible states is available and should be used going forward; Please use it on the above automations or scripts to fix this issue."
     },
+    "deprecated_command_actions": {
+      "title": "The command related actions are deprecated in favor of the new buttons",
+      "description": "`pause_program` and `resume_program` actions have been deprecated in favor of new buttons entities, if the command is available for your appliance."
```

Comment:
```suggestion
      "description": "`pause_program` and `resume_program` actions have been deprecated in favor of new button entities, if the command is available for your appliance."
```

Reply:
We should explain what the user should do to fix the issue, ie update any automations or scripts that are using this action and then press submit to fix the issue.